modules/rds/variables.tf

```diff
@@ -0,0 +1,106 @@
+#==============================================================
+# Database / outputs.tf
+#==============================================================
+
+#--------------------------------------------------------------
+# Network
+#--------------------------------------------------------------
+
+variable "database_subnet_cidr" {
+  default       = ["10.0.21.0/24", "10.0.22.0/24", "10.0.23.0/24"]
+  description   = "List of subnets to be used for databases"
+}
+
+variable "availability_zones" {
+  type          = "list"
+}
+
+variable "vpc_id" {}
+
+#--------------------------------------------------------------
+# Database
+#--------------------------------------------------------------
+
+variable "identifier" {
+	default     = "mydb-rds"
+	description = "Identifier for your DB"
+}
+
+variable "storage" {
+  default       = "180"
+  description   = "Storage size in GB"
+}
+
+variable "engine" {
+  default       = "postgres"
+  description   = "Engine type: e.g. mysql, postgres"
+}
+
+variable "engine_version" {
+  description   = "Engine version"
+
+  default {
+    postgres    = "9.6.5"
+  }
+}
+
+variable "instance_class" {
+  default       = "db.m4.xlarge"
+  description   = "aws instance"
+}
+
+variable "db_name" {
+  default       = "mydb"
+  description   = "Name of the first db"
+}
+
+variable "username" {
+  default       = "myuser"
+  description   = "User name"
+}
+
+variable "password" {
+  default       = "Password123"
```

Comment:
Please remove this, we don't want users to accidentally set un secure passwords.